src/modules.js

```diff
@@ -445,8 +445,13 @@ function exportRuntime() {
     // In pthreads mode, the following functions always need to be exported to
     // Module for closure compiler, and also for MODULARIZE (so worker.js can
     // access them).
-    ['PThread', 'ExitStatus', 'tempDoublePtr', 'wasmMemory', '_pthread_self',
-     'ExitStatus', 'tempDoublePtr'].forEach(function(x) {
+    var threadExports = ['PThread', 'ExitStatus', 'tempDoublePtr', '_pthread_self',
+    'tempDoublePtr'];
```

Comment:
indentation looks weird here - can be on the former line

Reply:
Fixed by removing the duplicate `tempDoublePtr` - how blind can one be  :D